packages/transaction-controller/src/TransactionController.ts

```diff
@@ -338,8 +344,10 @@ export class TransactionController extends BaseController<
       securityProviderRequest,
     }: {
       blockTracker: BlockTracker;
+      disableAdvancedGasFee: boolean;
```

Comment:
Did we forget to remove this one?

Reply:
Sorry removed here https://github.com/MetaMask/core/pull/1966/commits/26ef4aa8c7777b023882bd83e9385dcadadce07d